CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
 # ~~~
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.21)
```

Comment:
I think this is not going to work now in an env with older cmake. Keep minimum as it is?